web/modules/custom/hoeringsportal_public_meeting/assets/js/single-date.js

```diff
@@ -0,0 +1,10 @@
+require('../css/single-date.css')
+
+document.querySelectorAll('.date-to-element input.form-date')[0].disabled = true
+let tableRow = document.querySelectorAll('#field-pretix-dates-values tr')
+for (var i = 0; i < tableRow.length; i++) {
```

Comment:
[`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from) and [`Array.forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) are your friends: 

```js
const tableRows = Array.from(document.querySelectorAll('#field-pretix-dates-values tr'))
tableRows.forEach(tableRow => { … })
```